snmp/datadog_checks/snmp/data/default_profiles/ruckus-wap.yaml

```diff
@@ -5,7 +5,7 @@ sysobjectid:
   - 1.3.6.1.4.1.25053.3.1.4.*  # Ruckus Wireless
   - 1.3.6.1.4.1.25053.3.1.4.91  # Ruckus R510
 metrics:
-  - MIB: RUCKUS-UNLEASHED-SYSTEM-MIB
+  - MIB: RUCKUS-SYSTEM-MIB
```

Comment:
Why this change?

Reply:
line 12 have `RUCKUS-UNLEASHED-SYSTEM-MIB` value: https://github.com/DataDog/integrations-core/pull/15668/files#diff-bdaca85647c3553945c2c1ea7df2c8254bb0be0f361ff7aa6612a59144898073R12